scalyr_agent/json_lib/parser.py

```diff
@@ -729,17 +739,17 @@ def __peek_next_non_whitespace(self):
 
         while True:
             # TODO: support any Unicode / UTF-8 whitespace sequence.
-            raw_c = self.__scanner.peek_next_ubyte(none_if_bad_index=True)
-            if raw_c is None:
-                return raw_c
-            c = ord(raw_c)
-            if c == 32 or c == 9 or c == 13 or c == 10:
+            c = self.__scanner.peek_next_ubyte(none_if_bad_index=True)
```

Comment:
We should probably change the name for `peek_next_ubyte` to `peek_next_uchar`, right?  Same with `read_ubyte`.

Reply:
Yes, absolutely correct. Thanks.
Maybe we should rename the whole class?

Reply:
Yes, let's change the class name.